src/primer-api/index.ts

```diff
@@ -1,3 +1,2 @@
 export * from './primer-api';
 export * from './model';
-
```

Comment:
Any idea why this churn keeps happening?

Reply:
No.